Orange/canvas/report/tests/test_report.py

```diff
@@ -36,13 +35,15 @@
 def get_owwidgets(top_module_name):
     top_module = import_module(top_module_name)
     widgets = []
-    for root, dirs, files in os.walk(top_module.__path__[0]):
+    for root, _, files in os.walk(top_module.__path__[0]):
         root = root[len(top_module.__path__[0]):].lstrip(os.path.sep)
         for file in files:
             if file.lower().startswith('ow') and file.lower().endswith('.py'):
-                module_name = top_module_name + '.' + os.path.join(root, file).replace(os.path.sep, '.')[:-len('.py')]
+                module_name = top_module_name + '.'\
```

Comment:
I would rather have a too long line that a line that was split in a weird way just to make lint happy.

If I had to split this line, I would go with either
```
module_name = "{}.{}".format(
    top_module_name,
    os.path.join(root, file).replace(os.path.sep, '.')[:-len('.py')])
```
or
```
file, _ = os.path.splitext(file)
module_name = os.path.join(root, file).replace(os.path.sep, ".")
module_name = "{}.{}".format(top_module_name, module_name)
```

Reply:
Ok, fixed.